### DISCUSSION TITLE
Error: HTTP 404 fetching XXX.genome.fa.fai

### DISCUSSION THREAD

**Discussion by dtx282000** [GMOD/jbrowse-components#4379] [General]
Hi,Dr.  I want to run the JBrowse2 on my website using React. Here, an error occurred while I was running the program. So, how can I solve it?
![Uploading image.png…]()

**Comment by cmdcolin** [comment]
you may need to run "samtools faidx yourfile.fa" to generate the fai file (this is a so-called fasta index)
